docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,78 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
```

Comment:
These instructions are only for macOS. Should we call that out in this title? Unless you have additional task for Windows or Linux?

Reply:
macOS and Linux are in this iteration... Windows... haven't thought about that yet. The steps should be similar... close-ish... I'll test and write up Windows in an update.